pom.xml

```diff
@@ -54,14 +54,11 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.6</maven.compiler.source>
-        <maven.compiler.target>1.6</maven.compiler.target>
-        <envinject.lib.version>1.25-SNAPSHOT</envinject.lib.version>
+        <envinject.api.version>1.0-SNAPSHOT</envinject.api.version>
```

Comment:
@oleg-nenashev Why are you doing this for a *single* use of the dependency?  That makes no sense.  🐜  🐜  🐜